src/app/[locale]/dashboard/change-password/page.tsx

```
@@ -0,0 +1,21 @@
+import ChangePasswordForm from "@components/shared/change-password-form";
+import { useTranslations } from "next-intl";
+import React from "react";
+
+
+export default function page() {
+    // Translation
+    const t = useTranslations("change-password");
```

ESLint

[error] React Hook "useTranslations" is called in function "page" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)